lib/client/api_matrix.es6.js

```diff
@@ -11,12 +11,13 @@ foam.CLASS({
     to CSV format string or JSON matrix that can be displayed on HTML as
     a nested table.`,
   requires: [
-    'org.chromium.apis.web.Browser',
-    'org.chromium.apis.web.WebInterface',
-    'org.chromium.apis.web.BrowserWebInterfaceJunction',
-    'foam.dao.EasyDAO',
+    'foam.dao.AnonymousSink',
```

Comment:
Ah, I think I ended up in the foam repo instead of the foam2 repo by following links from the first search result for FOAM framework.